### PR TITLE
[1.13.x] RHPAM-4680 maven rpm module should include Red Hat repositories

### PR DESCRIPTION
**Pull request by radtriste**
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO|RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

- <b>(Re)run Jenkins tests</b>  
  Please add comment: <b>Jenkins [test|retest] this</b>

- <b>Prod tests</b>  
  Please add comment: <b>Jenkins (re)run [prod|Prod|PROD]</b>
</details>

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by spolti — APPROVED** [review]

File: modules/kogito-maven/3.8.x-rpm/configure [hunk 1]
```diff
@@ -7,3 +7,241 @@ SCRIPT_DIR=$(dirname "${0}")
 mkdir "${KOGITO_HOME}"/.m2
 cp -v "${SCRIPT_DIR}"/maven/* "${KOGITO_HOME}"/.m2
 cp -v "${SCRIPT_DIR}"/added/* "${KOGITO_HOME}"/launch/
+
```
Comment (MarianMacik) [inline modules/kogito-maven/3.8.x-rpm/configure:10]: Isn't everything twice here?

**Comment by kie-ci1** [comment]
**PR job** `#40` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-images/40/display/redirect

**Test results:**
- PASSED: 53
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-images/40/testReport/features.image.kogito-builder/kogito-builder image tests/Verify_that_the_Kogito_Quarkus_Serverless_Workflow_Extension_is_building_the_service_properly/">features.image.kogito-builder.kogito-builder image tests.Verify that the Kogito Quarkus Serverless Workflow Extension is building the service properly</a></summary>
S2I build failed, check logs!
</details>

File: modules/kogito-maven/3.8.x-rpm/configure (hunk 1 above)
Comment (radtriste) [reply modules/kogito-maven/3.8.x-rpm/configure:10]: yes you are right ...

**Review by rgdoliveira — APPROVED** [review]

**Comment by kie-ci1** [comment]
**PR job** `#41` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-images/41/display/redirect

**Test results:**
- PASSED: 53
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x/job/pullrequest/job/kogito-images/41/testReport/features.image.kogito-builder/kogito-builder image tests/Verify_that_the_Kogito_Quarkus_Serverless_Workflow_Extension_is_building_the_service_properly/">features.image.kogito-builder.kogito-builder image tests.Verify that the Kogito Quarkus Serverless Workflow Extension is building the service properly</a></summary>
S2I build failed, check logs!
</details>

**Pull request merged**